docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +296,46 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+There is a known issue that the computing power of modern hardware is growing faster than memory. Using more and more modern HW CPUs, we may never reach the theoretical peak performance of some models due to the memory bound problem. `Sparse weights decompression feature` allows to achieve greater performance on such models just due to more reasonable memory consumption.
+
+Sparse matrix is a matrix in which most of the elements are zero. To use memory more efficiently, we can store only the values of non-zero elements of such matrices in memory using special storage structures (e.g., CSC - Compressed Sparse Column or CСS - Compressed Column Storage). Using this, we can reduce the memory load and accordingly achieve higher performance on the models.
+
+`Sparse weights decompression feature` allows to pack weights for FullyConnected layers directly in the CPU plugin at the model compilation stage and store non-zero values in a special packed format. Then, during the execution of the model, the weights are unpacked and used in the computational kernel.
+
+To use this feature, the user is provided with property `sparse_weights_decompression_rate`, which can take values from the interval [0, 1]. `sparse_weights_decompression_rate` is the minimum fraction of nulls above which we use weights compression. The default value is `1`, which means the option is disabled.
+
```

Comment:
I think it would be useful to explain why the feature is disabled by default and what are the consequences when it's enabled (assume there is some overhead for non-sparse models).

Reply:
thanks, fixed, added explanation in NOTE section